src/redux/category/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  underConstruction: false,
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
```

Comment:
### Kindly implement a reducer for the categories that checks the status as it's part of project requirement 👍 
<img width="1007" alt="Screenshot 2023-06-14 at 2 34 11 PM" src="https://github.com/martinkarugaba/react-bookstore/assets/83048208/03bd7084-2ca9-4b19-ab17-b7deea11c380">